Vincent/Screens/ArtItem/ArtItemCategoryView.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  ArtItemCategoryViewController.swift
+//  Vincent
+//
+//  Created by 김연호 on 2022/10/12.
+//
+
+import UIKit
+
+import SnapKit
+import Then
+
+class ArtItemCategoryView: UIView {
+
+    private enum LayoutConstant {
+        static let spacing: CGFloat = 10.0
+        static let cellWidth: CGFloat = 90
+        static let cellHeight: CGFloat = 40
+    }
+
+    private let categoryDetailData = ["마네킹", "염료", "천", "실바늘", "재봉틀", "미싱기"]
+
+    private let categoryTitleLabel = UILabel().then {
+        $0.font = .systemFont(ofSize: 24, weight: .heavy)
+        $0.text = "섬유 패션"
+    }
+
+    private let collectionViewFlowLayout = UICollectionViewFlowLayout().then {
+        $0.minimumLineSpacing = 8
+        $0.itemSize = CGSize(width: LayoutConstant.cellWidth, height: LayoutConstant.cellHeight)
+        $0.sectionInset = UIEdgeInsets(top: LayoutConstant.spacing, left: LayoutConstant.spacing, bottom: LayoutConstant.spacing, right: LayoutConstant.spacing)
+        $0.scrollDirection = .horizontal
+    }
+
+    private lazy var categoryListCollectionView = UICollectionView(frame: .zero, collectionViewLayout: collectionViewFlowLayout).then {
+        $0.register(ArtItemCategoryCell.self, forCellWithReuseIdentifier: ArtItemCategoryCell.identifier)
```

Comment:
p5
```suggestion
        $0.register(ArtItemCategoryCell.self, forCellWithReuseIdentifier: ArtItemCategoryCell.className)
```
`NSObject+Extension` 파일에서, 모든 클래스를 ~.className 으로 추출할 수 있게 해놓았습니다.